### DISCUSSION TITLE
Rules are not finding any transactions with trigger 'Amount is greater than 0'

### DISCUSSION THREAD

**Discussion by dimaj** [firefly-iii/firefly-iii#6752] [Firefly III] [answered]
Just like the title says, when I have a rule trigger `Amount is more than... 0`, no transactions are found. When I change `0` to `0.00`, transactions are being found and rules are processing them.

Is this expected behavior?

Thanks!

**Comment by JC5** [comment, marked as answer]
That looks like a bug, nice find!

**Comment by dimaj** [comment]
Cool!

Should I open an issue on GH?

**Reply by JC5** [reply]
Just did, see https://github.com/firefly-iii/firefly-iii/issues/6753

It looks like it may be part of the underlying library.

**Reply by dimaj** [reply]
You already have a fix?!
You work fast!

Really appreciate your responsiveness.
